utils/manifests/definitions.go

```diff
@@ -0,0 +1,29 @@
+package manifests
+
+const template = "{\"apiVersion\":\"core.oam.dev/v1alpha1\",\"kind\":\"WorkloadDefinition\",\"metadata\":{\"name\":\"\"},\"spec\":{\"definitionRef\":{\"name\":\"\"}}}"
```

Comment:
We  could define one constant and using it in many places, that can easy to update the code when we bump the `crd` resource version.